src/main/webapp/script.js

```diff
@@ -198,57 +96,72 @@ function createCanvasForId(id) {
 }
 
 /**
- * Function to blur image and put on page
- * original and blurred images
- * @param {File} imageFile
+ * Updates the page with new uploaded image.
+ * It calls blurring method, puts new and blurred
+ * images on canvases and updates blurRadius input
+ * bar for new image.
+ * @param {String} imageUrl
+ * @param {Array<Rect>} blurAreas
+ * @return {Promise<void>} nothing
  */
-async function putImageOnPageAndBlur(imageFile) {
-  const imageUrl = URL.createObjectURL(imageFile);
-
-  // create Image object from url to put it on canvas.
-  const imageObj = new Image();
-  imageObj.src = imageUrl;
+async function processImage(imageUrl, blurAreas) {
+  const imageObj = await getImageFromUrl(imageUrl);
 
-  // need to wait until image loads to put it anywhere.
-  imageObj.onload = async () => {
-    const inputCanvas = document.getElementById('input-canvas');
-    drawImageOnCanvas(imageObj, inputCanvas);
+  const inputCanvas = document.getElementById('input-canvas');
+  const outputCanvas = document.getElementById('output-canvas');
 
-    const blurAreas = await getBlurAreas(imageFile);
+  // set blurRadiusInput bar max and default values.
+  const blurRadiusInput = document.getElementById('blurring-radius');
 
-    const blurredImage = getImageWithBlurredAreas(blurAreas, imageObj);
+  blurRadiusInput.setAttribute('max', Math.ceil(
+      getAverageRectSize(blurAreas) / (100 * 100) * 12 * 2));
```

Comment:
I think it's better if you made a constant for `(100 * 100) * 12 * 2)` and shortly explain it.